devtools/aesh/src/main/java/io/quarkus/cli/commands/ListExtensionsCommand.java

```diff
@@ -26,7 +26,7 @@
     @Option(shortName = 'a', hasValue = false, description = "Display name, group-id, artifact-id and version (default behaviour)")
     private boolean all = false;
 
-    @Option(shortName = 'f', hasValue = true, description = "Display in simplified format, only display the extension names or full format")
+    @Option(shortName = 'f', hasValue = true, description = "Display only extension names with name value, in simplified format with simple value or full format.")
```

Comment:
I believe there is a typo. What does `name value` mean? 

What about: `Switch between the concise and full output. The concise output only displays the extension name`.

Reply:
I am not very good in english, i must stop to PR dor documentation but, I just want to say there 3 format: name, simple and full. The name only display extension name, the simple add a human readable label and the full add the version detection